### PR TITLE
Add support for systemd

### PR DESCRIPTION
**Pull request by corny**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/ginsys/ansible-role-seafile/pull/20" target="_blank">ReviewNinja

File: tasks/9_init.yml [hunk 1]
```diff
@@ -9,25 +9,35 @@
...
-- name:         provision init script
+- name:         provision sysvinit script
   template:
     src:        init/{{ item.name }}.initd.{{ ansible_os_family }}
     dest:       /etc/init.d/{{ item.name }}
     owner:      root
     group:      root
     mode:       0750
-  when:         ansible_os_family in item.os_family
+  when:         ansible_service_mgr == "sysvinit"
```
Comment (srgvg) [inline tasks/9_init.yml]: You here remove the logic conditional handling `SEAFILE_INIT_SCRIPTS` where Red Hat has two sysv init scripts, and Debian only one. Is that intentionally? I'd expect this should be
`ansible_os_family in item.os_family and ansible_service_mgr == "sysvinit"`

Comment (corny) [reply tasks/9_init.yml]: Ich checked the value of `SEAFILE_INIT_SCRIPTS`. Why is the seahub startup script only installed on RedHat?

Comment (srgvg) [reply tasks/9_init.yml]: IIRC, when I started out this role (seafile 4.x) that was how the docs explained what needed to be done, and those were different scripts. It is quite possible that this changed over time, and I can't find anything similar any more. seafile.sh and seahub.sh will probably have evolved, too, and shoudl perhaps be updated in the role, or the logic rewritten as that the files deployed from the tarball are used to update them.

Comment (srgvg) [reply tasks/9_init.yml]: (As the sysv init scripts in the role are still the versions from back then)
Is this something you'd willing to update as well?

Comment (corny) [reply tasks/9_init.yml]: I read the https://manual.seafile.com/deploy/start_seafile_at_system_bootup.html and now I understand that RedHat requires two scripts and Debian only one. I updated my pull request.

**Review by srgvg — APPROVED** [review]

**Comment by srgvg** [comment]
Thank you!

**Pull request merged**